### PR TITLE
Add user reason code EndpointNotReachable

### PR DESCRIPTION
**Pull request by prankulmahajan**

### CODE REVIEW

File: pkg/messages/messages_en.go [hunk 1]
```diff
@@ -95,6 +95,18 @@ var messagesEn = map[string]Message{
 		Type:        codes.InvalidArgument,
 		Action:      "Please check all node's labels by using kubectl command",
 	},
+	EndpointNotReachable: {
+		Code:        EndpointNotReachable,
+		Description: "Endpoint is incorrect",
+		Type:        codes.PermissionDenied,
+		Action:      "User should review the storage secret endpoint",
```
Comment (prankulmahajan) [inline pkg/messages/messages_en.go:102]: add the endpoint

Comment (prankulmahajan) [reply pkg/messages/messages_en.go:102]:
```
Verify if iks_token_exchange_endpoint_private_url is reachable from the cluster. This field can be found from the storage-secret-store secret from kube-system namespace.
```

File: pkg/messages/messages_en.go [hunk 2]
```diff
@@ -95,6 +95,18 @@ var messagesEn = map[string]Message{
...
+		Code:        EndpointNotReachable,
+		Description: "Endpoint is incorrect",
+		Type:        codes.Unavailable,
+		Action:      "Verify that iks_token_exchange_endpoint_private_url is reachable from the cluster. You can find this url by running 'kubectl get secret storage-secret-storage -n kube-system'.",
+	},
+	Timeout: {
+		Code:        Timeout,
+		Description: "Timed out for response from endpoint",
+		Type:        codes.DeadlineExceeded,
+		Action:      "Wait for a few mninutes and try again. If the error persists user can open a container network issue.",
```
Comment (sameshai) [inline pkg/messages/messages_en.go:108]: Change minutes typo

**Review by arahamad — APPROVED** [review]
approved

**Review by sameshai — APPROVED** [review]

**Pull request merged**